api/migrations/config.js

```diff
@@ -13,10 +14,9 @@ const RUN_IN_PROD = !!process.env.AWS_RETHINKDB_PASSWORD;
 
 if (RUN_IN_PROD && process.argv[4] === 'down') {
   throw new Error('Do not drop the production database!!!!!');
-  process.exit(1);
```

Comment:
Why is this line being removed?

Reply:
Eslint says this is unreachable code. https://eslint.org/docs/rules/no-unreachable

Reply:
Ah, fair enough!